src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/account/BceidAccountApiDelagateImpl.java

```diff
@@ -1,27 +1,49 @@
 package ca.bc.gov.open.jag.efilingapi.account;
 
+import ca.bc.gov.open.bceid.starter.account.BCeIDAccountService;
+import ca.bc.gov.open.bceid.starter.account.GetAccountRequest;
+import ca.bc.gov.open.bceid.starter.account.models.IndividualIdentity;
+import ca.bc.gov.open.jag.efilingapi.account.service.BceidAccountMapper;
 import ca.bc.gov.open.jag.efilingapi.api.BceidAccountApiDelegate;
 import ca.bc.gov.open.jag.efilingapi.api.model.BceidAccount;
+import ca.bc.gov.open.jag.efilingapi.utils.SecurityUtils;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
 import javax.annotation.security.RolesAllowed;
+import java.util.Optional;
 import java.util.UUID;
 
 @Service
 public class BceidAccountApiDelagateImpl implements BceidAccountApiDelegate {
 
+
+    private final BCeIDAccountService bCeIDAccountService;
+    private final BceidAccountMapper bceidAccountMapper;
+
+    public BceidAccountApiDelagateImpl(BCeIDAccountService bCeIDAccountService, BceidAccountMapper bceidAccountMapper) {
+        this.bCeIDAccountService = bCeIDAccountService;
```

Comment:
question:
I thought we dont want back to back capital letters?
```bCeIdAccountService``` is better right?

Reply:
comming from the `BCeID` ...